### PR TITLE
Adds More species to the charged Black extract

### PR DESCRIPTION
**Pull request by boodaliboo**
Before this pull request you could only get the round start races from this crossbreed which felt unfair as making atleast one of these could take around a hour atleast. so i made this pull request to fix it

🆑
rscadd: New species have been added to the charged black slime extract!
/:cl:

### CODE REVIEW

**Comment by Jc1469** [comment]
how about not giving xeno more stuff thanks

**Comment by boodaliboo** [comment]
> how about not giving xeno more stuff thanks

and why is that? most xeno bio items take most of the round to get the only thing thats really unbalanced is gold slimes

**Comment by drderp3635** [comment]
Might I point out to anyone who doesn't know how godawful inefficient charged black slimes are. You use 10 yellow extracts to an adult black, then you have to put 10 plasma in it. Then you get one random race roll, some of which are just awful. The actual amount of work you would have to put forward to actually get a specific race, is far more than what it is worth.

File: code/modules/mob/living/carbon/human/species_types/snail.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	speedmod = 6
 	punchdamagehigh = 0.5 //snails are soft and squishy
 	siemens_coeff = 2 //snails are mostly water
-	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | RACE_SWAP
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | RACE_SWAP | SLIME_EXTRACT
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/snail.dm]: No

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/snail.dm (hunk 1 above)
Comment (boodaliboo) [reply code/modules/mob/living/carbon/human/species_types/snail.dm]: how do i change it to that?

Comment (boodaliboo) [reply code/modules/mob/living/carbon/human/species_types/snail.dm]: i think i did it

File: code/modules/mob/living/carbon/human/species_types/snail.dm [hunk 2]
```diff
@@ -12,7 +12,7 @@
 	speedmod = 6
 	punchdamagehigh = 0.5 //snails are soft and squishy
 	siemens_coeff = 2 //snails are mostly water
-	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | RACE_SWAP
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | RACE_SWAP
```
Comment (monster860) [inline code/modules/mob/living/carbon/human/species_types/snail.dm]: AAAA YOU ADDED A SPACE AT THE END

**Review by monster860 — CHANGES_REQUESTED** [review]

**Pull request merged**